### DISCUSSION TITLE
Add Download option in table

### DISCUSSION THREAD

**Discussion by muksidin** [Budibase/budibase#8730] [Ideas]
Hello, 
How about add fiture Download option in table, so with this fiture we can download some format documents such as CSV, JSON, PDF etc.
I think this fiture is enough important and i need this fiture.

**Comment by melohagan** [comment]
There is the **Export data** action which allows you to save the selected rows of a table as CSV.

**Reply by muksidin** [reply]
Yes if it's in developer mode, but my mean is table component in design.

**Reply by muksidin** [reply]
@melohagan ahh, i just understand if there is a action to export data. Okay thanks
